pkg/resources/dynamic_table.go

```diff
@@ -75,6 +75,34 @@ var dynamicTableShema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Specifies a comment for the dynamic table.",
 	},
+	"refresh_mode": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "INCREMENTAL if the dynamic table will use incremental refreshes, or FULL if it will recompute the whole table on every refresh.",
+		ForceNew:    true,
```

Comment:
With our upcoming resources redesign on the road to V1 we have a topic of force new versus create or replace. According to the docs (both for `refresh_mode` and `initialize`):
```
To modify the property, replicate the dynamic table with a CREATE OR REPLACE DYNAMIC TABLE command.
```
So this opens two possibilities:
1. We leave the `ForceNew` as is.
2. We modify the resource's logic to use `OrReplace` when refresh mode or initialize changes.

Both have their pros and cons. Because the discussion will still take place globally for every resource, I think there is no need to change it here. My preference is to use OrReplace, though.

Reply:
Thanks for this context. It was also interesting to read through this codebase and see how `create or replace` is implemented for different resources.

Since you say this will be more of a global decision, I will leave it as is to save myself the time of getting the alternative working, but I'll check back in the future to see where things ended up.